conf/webpack/webpack.config.dev.js

```diff
@@ -19,7 +19,7 @@
 const webpack = require('webpack');
 const config = require('./webpack.config');
 
-config.devtool = 'eval';
+config.devtool = 'inline-source-map';
```

Comment:
Possible values (for the version of webpack we use, v`1`), [listed here](https://github.com/webpack/docs/wiki/configuration#devtool).